www/getlisted.mako

```diff
@@ -0,0 +1,117 @@
+<%inherit file="_base.mako"/>
+<%def name="title()">Journal Outreach</%def>
+<%def name="description()">Get more journals listed as eligible for the Preregistration Challenge.</%def>
+
+<%def name="navigation()">
+    ${self.navlinks('prereg')}
+</%def>
+
+<%def name="stylesheets()">
+
+    <!-- BEGIN PAGE LEVEL PLUGIN STYLES -->
+    <link href="../static/plugins/fancybox/source/jquery.fancybox.css" rel="stylesheet" />
+    <link href="//cdn-images.mailchimp.com/embedcode/classic-081711.css" rel="stylesheet" type="text/css">
+    <style type="text/css">
+        #mc_embed_signup{
+            background:#e7f5fc;
+            clear:left;
+            font:14px 'Open Sans';
+            margin-bottom: 30px;
+        }
+        #mc_embed_signup form {
+            padding: 5px;
+        }
+
+        #mc_embed_signup_scroll .prereg-sign-up {
+            padding: 15px;
+            background-color: #2f89af;
+            padding: 15px;
+            color: white;
+        }
+
+        #mc_embed_signup h2 {
+
+            color: white;
+            margin-top: 0px;
+            margin-bottom: 0px;
+            font-weight: 400;
+            font-size: 21px;
+        }
+
+        div.mc-field-group {
+            background-color: white;
+            padding: 3%;
+            margin-top: 2px;
+            margin-bottom: 2px;
+            width: 100% !important;
+        }
+
+        #mc_embed_signup input {
+            border: none;
+        }
+
+        #mc_embed_signup .button {
+            background-color: #2f89af;
+            display: block;
+            margin: 5px auto;
+        }
+
+        #mc_embed_signup div .mce_inline_error {
+            margin-bottom: 0px !important;
+        }
+    </style>
+    <!-- END PAGE LEVEL PLUGIN STYLES -->
+
+</%def>
+
+
+<!-- BEGIN PAGE CONTAINER -->
+<div class="page-container pre-reg">
+
+    <!-- BEGIN CONTAINER -->
+    <div class="container min-height margin-top-50 col-md-12">
```

Comment:
The `min-height` and `margin-top-50` classes don't appear to have any effect. Did you mean to do `style="margin-top: 50px`? @evomellor Can you run this locally and ensure that this page appears the way you intend it to?